internal/client/network/config/config.go

```diff
@@ -0,0 +1,16 @@
+// Copyright 2025 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package config
+
+import (
+	"github.com/ChainSafe/gossamer/internal/client/network/types/multiaddr"
+	peerid "github.com/ChainSafe/gossamer/internal/client/network/types/peer-id"
+)
+
+// MultiaddrPeerId is the address of a node, including its identity.
+// This struct represents a decoded version of a multiaddress that ends with "/p2p/<peerid>".
+type MultiaddrPeerId struct {
```

Comment:
why have separate package config for this?

Reply:
That's where it's found in substrate ([link](https://github.com/paritytech/polkadot-sdk/blob/7f905e28ad88bdfbc679e70f7bfc97546515e627/substrate/client/network/src/config.rs#L142)).  There will be more config associated types added to this package in future work.